### ISSUE TITLE
Policy regarding flagging packages

### ISSUE THREAD

**Issue by dandv** [MeteorCommunity/discussions#20]
From @raix:

> We should probably have a policy about flagging packages
> When official package is published, ask wrapper maintainers to deprecate their packages:
> 1. Owner agrees - They should probably do `meteor admin set-unmigrated` instead?
> 2. Owner doesn't agree - We should respect that and figure out why
> 3. Owner cant be reached/doesn't respond within X weeks of contact - we could assume that it's inactive - flagging would make sense?

**Comment by dandv** [comment]
Give that I wrote the [Atmosphere police](https://github.com/percolatestudio/atmosphere/issues/297) post, I think we should be more effective in cleaning up the pollution in our Atmosphere. In my estimate, between 25% and 50% of the client and client/server packages there are junk in one way or another (abandoned, outdated, badly migrated). I haven't looked at server-side packages yet.

I wish there was a way to _remove packages altogether_, at least from counts. "2926 packages", as meteor.com boasts, is highly misleading, and may make us look like we're artificially inflating the number of packages, and maybe even their quality ("Atmosphere is the catalog of the most popular, trusted, and reliable packages").

Re. flagging, I do hope Percolate can implement the [moderator privileges](https://github.com/percolatestudio/atmosphere/issues/297#issuecomment-63397971), where clearly broken packages can be flagged faster, yet still require at least two or three votes, to prevent abuse.

> Owner doesn't agree - We should respect that and figure out why

In my experience, most owners have agreed, but a few refused, for reasons they couldn't quite justify. There is emotional attachment to one's package, that can't be negated. There's also the English language barrier in some case.

Fact is, if as a result of #14 we get official Meteor packages blessed by the very author of a library, with a passing test suite, an old wrapper that's much beloved by its author does nothing on Atmosphere but clutter it, and should be unmigrated by moderators, not just flagged. I see no use case in allowing clearly obsoleted packages to be featured in search results.

**Comment by dandv** [comment]
https://atmospherejs.com/mrt/momentjs is a clear example on when packages should be speed-removed. It shows in searches.

The problem is complicated by the fact that [the `meteor` tool doesn't want when installing unmigrated dependencies](https://github.com/MeteorCommunity/discussions/issues/17).

**Comment by awatson1978** [comment]
Perhaps there should be secondary counts.  

2926 packages
1252 verified packages
800 packages with unit tests
340 packages with acceptance tests

Then the counts wouldn't be misleading, and people wouldn't need to be such busybody's and so much in other people's business.  

> There is emotional attachment to one's package, that can't be negated. There's also the English language barrier in some case.

There's also differing definitions of quality and maintenance.  Some authors aren't supporting for community maintenance, but personal projects.  Some package authors may be in the process of forking a package, rather than maintaining a package.  

>  I see no use case in allowing clearly obsoleted packages to be featured in search results.

Digital archeology.  There's lots of knowledge and old snippets that can be gleaned from obsolete packages.  Here's a real-world example from the past two months: 

I've been working on a child day-care messaging app, which helps find nannys for parents on the go.  We've integrated with Google Contacts, and are currently using [percolate:google-api](https://atmospherejs.com/percolate/google-api) which is the most up-to-date and maintained package to connect to Google API.  Yet.  It absolutely did not have the code that we needed to get access to the contacts feed.  As a maintained package, it failed it's job.  It was the [long:google-contact](https://atmospherejs.com/long/google-contacts) package - which was broken - that had the pattern and documentation we needed to get things working.

Would it have been ideal for long:google-contact to be up to date?  Sure.  Would it have been better for that package to be hidden from searches?  No.  It might have been abandoned, but somebody had taken the time to integrate Google Contacts with Meteor.  And even though the package was broken, it was still relevant and had the code we needed to get our integration working.  It wasn't necessarily Long's fault that the package is broken.  MDG changed the underlying API.  

Honestly, the biggest problem wasn't that it was abandoned.  It was that it was in Coffeescript.  Perhaps we could extend the counts further:  

2926 packages
2201 Javascript packages
725 Coffeescript packages

**Comment by dandv** [comment]
Agree. Long's package wasn't "clearly obsoleted" though. It went to some trouble to integrate an API.

I'm talking about cases like extremely thin wrapper packages that just throw a package.js along a _copy_ (not even git submodule) of the original code.
